src/components/pastestack/PasteStackApp.js

```
@@ -0,0 +1,173 @@
+
+  const handleNewClip = useCallback((clip) => {
+    if (!isPaused) {
+      // Add a uniqueId for list key purposes, especially if items might have same db ID temporarily
+      // or if items are not from DB yet (though current flow sends DB items)
+      const newItem = { ...clip, uniqueId: `${clip.id}-${Date.now()}` };
+      setStackItems(prevItems => [newItem, ...prevItems.slice(0, MAX_STACK_ITEMS - 1)]);
+    }
+  }, [isPaused]);
+
+  useEffect(() => {
+    if (window.electron && window.electron.receive) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)